packages/app/src/DashboardPage.tsx

```diff
@@ -1086,7 +1137,8 @@ export default function DashboardPage({
                 />
                 <div style={{ width: 200 }} className="ms-2">
                   <GranularityPicker
```

Comment:
It'd be really helpful to have a tooltip here so that users know that granularity is locked to auto when live mode is enabled, otherwise it's a bit confusing why the dropdown isn't working.

Reply:
good call, added a tooltip:

![Screenshot 2024-07-11 at 3 54 36 PM](https://github.com/user-attachments/assets/b6a07bda-4291-4e8d-be2a-a8257b6362b3)